vscode/src/utils.ts

```diff
@@ -405,3 +407,11 @@ export async function revealDevice(ctx: Context, hwid: string): Promise<void> {
   }
   await ctx.getDeviceView()?.reveal(device, { "focus": true, "select": false, "expand": true });
 }
+
+export async function listPackages(ctx: Context): Promise<Package[]> {
+  const { stdout } = await toitExecFilePromise(ctx, "pkg", "list");
+  return stdout.split("\n").
```

Comment:
Alternatively, probably better, is to change the CLI to emit everything in one big json.
In theory there is nothing preventing the json output to be nicely formatted (with multiple lines), and that would fail then.

Reply:
This is a problem throughout the extension. It works (for now).